build/includes/sdk.mk

```diff
@@ -28,6 +28,7 @@ grpc_release_tag = v1.16.1
 sdk_build_folder = build-sdk-images/
 SDK_FOLDER ?= go
 COMMAND ?= gen
+SDK_IMAGE_TAG=$(build_sdk_prefix)$(SDK_FOLDER):$(VERSION)
```

Comment:
We have a cross platform SHA function we can use, for example:
https://github.com/googleforgames/agones/blob/master/build/includes/sdk.mk#L23

This will mean the image only gets recreated on Dockerfile change, rather than on every commit.

Reply:
Yes that's true, but we should add sha of bash scripts also. Will update this PR.

Reply:
We tend to fudge that a little with the `# forceUpdate` comment in the main build Dockerfile - it works well enough for now :man_shrugging: maybe there is a better way?

Reply:
I will investigate options tomorrow.
shasum function can work on multiple files, we can make equivalent variables for other platforms:
```
shasum ./build-sdk-images/cpp/* | shasum
```
would compute a hash of hashes

Reply:
QQ - does this mean we'll have two tags for the sdk image, one by hash, ,and one by commit?

(Was working on conformance tests for Reserve, and I think I ran into this issue)

Reply:
Hello, I think we can have only one tag - hash of all sh files as an ideal solution without hash of commit.
There was an issue with `make run-sdk-conformance-test SDK_FOLDER=rust`

Reply:
Sorry - this is for the build image - not the sdk-server image. All good. Ignore me.